code/game/objects/items/devices/chameleonproj.dm

```diff
@@ -207,15 +207,12 @@
 
 /obj/item/borg_chameleon/proc/toggle(mob/living/silicon/robot/syndicate/saboteur/user)
 	if(active)
-		playsound(src, 'sound/effects/pop.ogg', 100, 1, -6)
-		to_chat(user, "<span class='notice'>You deactivate [src].</span>")
-		deactivate(user)
+		to_chat(user, "<span class='notice'>You reconfigure [src].</span>")
+		activate(user)
 	else
```

Comment:
could you add an early return here?

Reply:
applied

Reply:
I think you applied it to another proc. This one still could use an early return

Reply:
I'll check when I get home (eta 6 hours)

Reply:
Yep I missed it, applied